### PR TITLE
devel

### PR DESCRIPTION
**Pull request by ijiraq**
Makes use of DATASEC to set image scaling.

### CODE REVIEW

**Comment by austinbeauch** [comment]
Broke a few things, I checked out the branch locally and will work with it.

File: src/daomop/storage.py [hunk 1]
```diff
@@ -1348,11 +1345,15 @@ def isfile(uri):
...
+    """Copy a file to/from VOSpace. With upto 10 retries on errors,
+
+    :return: content disposition value from data service
+    :rtype: basestring
+    """
     count = 1
     while True:
         try:
-            return vospace.client.copy(source, destination)
+            return vospace.client.copy(source, destination, disposition=True)
```
Comment (austinbeauch) [inline src/daomop/storage.py:1356]: I made a comment here but I think it wasn't on the pull request so I don't know if you'll see it... but disposition isn't a parameter for vospace.client.copy.

**Comment by austinbeauch** [comment]
After removing `disposition=True` from that line in storage there's a new error that's popping up when trying to load the candidate for rendering. Line 729 of viewer.py in loaded_hdu. Something is going wrong with `self.image_list[key].get()` as I'm continually getting a TypeError `expected string or buffer.` Any idea what could be creating this?

The call to loaded_hdu comes from 524 of viewer, at `image.load_hdu(self.loaded_hdu)`

**Comment by ijiraq** [comment]
Disposition is a parameter of the prerelease version of the vos client.   You can upgrade using

pip install vos --upgrade --pre

JJ

**Comment by austinbeauch** [comment]
Never mind my other comment about the string or buffer. After the vos update that hasn't happened again. Now I'm dealing with a key error coming from clear_candidate_images which I should be able to work out tomorrow.

**Pull request merged**